### DISCUSSION TITLE
If some of the libraries in my project still have the old architecture, can I upgrade to a new arch?

### DISCUSSION THREAD

**Discussion by YOEL311** [reactwg/react-native-new-architecture#145] [Q&A] [answered]
I use any library that hasn't been upgraded to the new architecture 
Is it possible to upgrade my project to a new arch?
Thanks

**Comment by thymikee** [comment, marked as answer]
Yes, you can use the interop layer. Read more here: https://github.com/reactwg/react-native-new-architecture/discussions/135

**Comment by flokol120** [comment]
Maybe take a look at the _Interop Layer_, If you haven't seen it before: #135 (whoops, too slow...)
